docs/general/serializers.md

```diff
@@ -379,6 +379,31 @@ class PostsController < ActionController::Base
   end
 end
 ```
+Note that `view_context` is made available through `ActionController::Base`. The `view_context` will be `nil` when accessed from a controller that inherits from `ActionController::API`. In such a case, a helper will need to be instantiated as needed, e.g.:
```

Comment:
Isn't there anything else that might substitute for `view_context`? Maybe `helpers`? or just `self`?

Reply:
Long story short - I can remove the example I posted if it helps keep the documentation more clear and coherent. 

When I was working on this for my own project, I found that `Rails.application.helpers`  only included the helpers defined by the current application, (i.e., helpers you wrote), and does not include other view helpers, like `NumberHelper` or `CsrfHelper` etc. 

I only proposed a generic solution to imply that "some class that has these modules included needs instantiated, somehow" and that `view_context` wont include them. 

I understand if the example I provided is out of scope for the documentation, and also something you'd rather not officially represent. Would it be simpler if I just removed the example, and only left the note itself? I.e.,

```ruby
Note that `view_context` is made available through `ActionController::Base`. The `view_context` will be `nil` when accessed from a controller that inherits from `ActionController::API`.
```

Reply:
How about we edit anywhere on the page that says view_context (for subclasses of ActionController:Base), and adds another example or note that any controller reference which provides the desired scope is acceptable, for example `:self` may be useful for subclasses of `ActionController::API` which does not include ActionView::ViewContext ?

Thoughts?

Reply:
Im not sure if I follow the use of `:self`, as it too is always `nil` for:

```ruby
class SomeApiController < ActionController::API
  serialization_scope :self
end
```

I think I can change the wording to match something similar to what you have above, though.